content/applications/services/helpdesk/overview/reports.rst

```diff
@@ -1,60 +1,188 @@
-============================
-Reports for a Better Support
-============================
+=========
+Reporting
+=========
 
-An efficient customer service solution should have a built-in reporting option. Reports allow you to
-track trends, identify areas for improvement, manage employees’ workloads and, most importantly,
-meet your customer’s expectations.
+Odoo's :guilabel:`Helpdesk` app includes several reports that provide the opportunity to track
+trends for customer support tickets, identify areas for improvement, manage employee workloads,
+and confirm when customer expectations are met.
 
-Cases
-=====
+Available reports
+=================
 
-Some examples of the reports Odoo Helpdesk can generate include:
+The following are the reports available in Odoo :guilabel:`Helpdesk`. To view the different reports,
+go to :menuselection:`Helpdesk --> Reporting`.
 
-- The number of tickets *grouped by* team and ticket type.
+Ticket Analysis
+---------------
 
-| In this manner, you are able to evaluate which ticket types have been the most frequent ones,
-  plus the workload of your teams.
-| Apply *Time Ranges* if you would like to make comparisons to a *Previous Period* or a *Previous
-  Year*.
+The :guilabel:`Ticket Analysis` report provides an overview of every customer support ticket in the
+database, including the number of tickets assigned among teams and individual users. This report is
+useful in identifying where teams are spending the most time, and determining if there is an uneven
+workload distribution among the support staff.
 
-.. image:: reports/report_team_ticket_type.png
+The default report counts the number of tickets per team and groups them by stage.
+
+.. image:: reports/reports-tickets-default.png
+   :align: center
+   :alt: View of Ticket Analysis report default view
+
+Alternative measures can be selected to track where the most time is spent at different points in
+the workflow. To change the measures used in the report, click :guilabel:`Measures` and choose from
+the drop-down menu. The available measures for the :guilabel:`Ticket Analysis` report include:
+
+- Average Hours to Respond
+- Hours Open
+- Hours Spent
+- Hours to Assign
+- Hours to Close
+- Hours to First Response
+- Hours to SLA Deadline
+- Rating /5
+- Count
```

Comment:
Since these are menu options, I believe the correct protocol would be to give each of these :guilabels:

Also, it might be beneficial if there was a screenshot taken of the open Measures menu, showing all these options, where the Measures menu is on the screen, and so on...

Just a thought...

Reply:
confirmed with sam, since it isn't a direction/command, they don't get guilabels, but they can be italicized, so I made that update👍

Reply:
@jero-odoo KC's suggestion is correct because this this is a UI walk-through that is a continuation of an active instruction (e.g. open the menu and then make a choice).

Here's a good example of what that looks like, where each UI element is given a guilabel and a small description after a colon, like this: https://github.com/odoo/documentation/pull/3532

To force the guilabel use, I would remove the previous sentence before the list and end the previous one with a semicolon:
```
To change the measures used in the report, click the :guilabel:`Measures` button, and choose an option from the drop-down menu:
```